pkg/controllers/manager.go

```diff
@@ -54,16 +59,22 @@ func NewManagerOrDie(config *rest.Config, options controllerruntime.Options) Man
 
 // RegisterControllers registers a set of controllers to the controller manager
 func (m *GenericControllerManager) RegisterControllers(controllers ...Controller) Manager {
-	for _, controller := range controllers {
-		controlledObject := controller.For()
-		var builder = controllerruntime.NewControllerManagedBy(m).For(controlledObject)
-		if namedController, ok := controller.(NamedController); ok {
+	for _, c := range controllers {
```

Comment:
can we go back to `controller` rather than `c`? We've used that style most other places

Reply:
It was originally `controller` but since I added in this package `"sigs.k8s.io/controller-runtime/pkg/controller"`, I had to rename the controller to c.

Reply:
ahh i c